packages/flutter_lints/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: flutter_lints
+description: Recommended lints for Flutter apps, packages, and plugins
+version: 0.1.0
+homepage: https://github.com/flutter/packages/tree/master/packages/flutter_lints
+publish_to: none
```

Comment:
I think you want to remove this?

Reply:
Ultimately, yes. However, while I pull in the "lints" dependency directly from github below, there's a lint telling me that I need this. 😝

Reply:
ha!